third_party/2and3/typing_extensions.pyi

```diff
@@ -24,6 +24,7 @@ Protocol: _SpecialForm = ...
 Final: _SpecialForm = ...
 def final(f: _F) -> _F: ...
 Literal: _SpecialForm = ...
+IntVar: object = ...
```

Comment:
Why is this typed as `object` when the implementation is a function?

Reply:
I patterned this typing after the entry in `typing.pyi` for normal `TypeVar`s. Just as with `TypeVar` there is no natural typing for these as they require special casing in the type checkers.  We could type this as a function if folks would like, but using it in a typechecker that hasn't implemented the backend functionality yet will still not work, so I'm not sure what adding that buys us.  Would folks prefer it as a 
```
def IntVar(__name: str) -> typing.Type
```
?

Reply:
I agree that it will probably not have any effect on type checkers, but let's type it correctly. After all, some other tools might use stubs as well.